range.go

```diff
@@ -111,8 +111,8 @@ func (r *RangeOp) Fields(fieldsToRead []string) *RangeOp {
 	return r
 }
 
-// Limit sets the number of rows returned per call. If not set, a default
-// value would be applied
+// Limit sets the number of rows returned per call. A limit must be provided
+// otherwise an error will be returned.
```

Comment:
This function doesn't return an error. Would it make more sense to say "A limit must be provided otherwise an error will be returned by the Range function."